config/environments/uat.rb

```diff
@@ -121,4 +121,6 @@
 
   # Google Tag Manager ID for EOSDIS usage stats
   config.tag_manager_id = 'GTM-WNP7MLF'
+
+  config.cmr_email_frequency = ENV['cmr_email_frequency'].to_i
```

Comment:
this variable needs to be documented in the OPS manual. Point it out to rosy please.